packages/debugger/src/handlers/editor.ts

```diff
@@ -82,37 +83,55 @@ export class EditorHandler implements IDisposable {
       this._addBreakpointsToEditor();
     });
 
+    this._debuggerService.model.breakpoints.clicked.connect((_, breakpoint) => {
+      this._selectedBreakpointLine = breakpoint.line ?? null;
+      this._selectedBreakpointPath = breakpoint.source?.path ?? null;
```

Comment:
As in my other comment, here I would also recommend storing the entire breakpoint:

```suggestion
      this._selectedBreakpoint = breakpoint;
```